### PR TITLE
Resolve issue #510.

### PR DESCRIPTION
**Pull request by mageekguy**
Mock asserter is now case insensitive.
Hope that this PR will be merged in master in less than a month, because i'm just losing an hour due to this bug ;).

### CODE REVIEW

**Comment by jubianchi** [comment]
:heart:

**Comment by mikaelrandy** [comment]
:+1: (once conflict on Changelog will be resolved)

**Comment by jubianchi** [comment]
I'm dealing with it ;)
will be merged soon :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,12 +5,18 @@
 * [#515](https://github.com/atoum/atoum/pull/515) Fix PHP7 support in the basic resolver ([@hywan])
 * [#516](https://github.com/atoum/atoum/pull/516) Add a "callStaticOnTestedClass" method ([@mikaelrandy])
 
-# 2.3.0 - 2015-10-22
+## Bugfix
+
+* [#526](https://github.com/atoum/atoum/pull/526) Mock asserter is not case sensitive ([@mageekguy])
+
+
+# 2.3.0 - 2O15-10-22
```
Comment (mikaelrandy) [inline CHANGELOG.md]: @jubianchi you have reintroduce the typo on the date : "2O15" (with the letter O) instead of "2015" (with digit)

Comment (jubianchi) [inline CHANGELOG.md]: #fail :dancers:

Comment (mikaelrandy) [inline CHANGELOG.md]: Why you haven't corrected this before merging ?

(fixed here : https://github.com/atoum/atoum/commit/12fa40738d417902738c9cd561868396523e8785)

Comment (jubianchi) [inline CHANGELOG.md]: oh sorry i misclicked :/ I had the fix locally :/

**Pull request merged**